packages/builder/src/pages/builder/portal/apps/onboarding/index.svelte

```diff
@@ -74,6 +78,10 @@
       const newPlusIntegrations = {}
 
       Object.entries($integrations).forEach(([integrationType, schema]) => {
+        // google sheets not available in self-host
+        if (helpers.isGoogleSheets(integrationType) && !cloudHosted) {
```

Comment:
Could change the `cloudHosted` check to use the new field for `googleDatasourceConfigured` from the organisation store. 

It won't change much considering it's v unlikely sheets will be configured but it'd be nice to be consistent. Examples where it would produce a more accurate result:

- A self host install that's running for the first time and has google env vars - likely QA 
- A cloud install that doesn't have the google env vars - likely QA or feature branch.